Dockerfile.template.erb

```diff
@@ -46,13 +46,13 @@ ENV TINI_VERSION=0.18.0
 <% if is_windows %>
 RUN powershell -Command "Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))"
 
-# Fluentd depends on cool.io which is only available for Ruby < 2.5, so need to install Ruby < 2.5 and install msys2 to get dev tools
-RUN choco install -y ruby --version 2.4.2.2 --params "'/InstallDir:C:\'" \
-&& choco install -y msys2 --params "'/NoPath /NoUpdate /InstallDir:C:\ruby24\msys64'" \
-&& refreshenv \
```

Comment:
Is changing `&&` to `RUN` command intentional?

Reply:
Yes,
The huge RUN command didn't handle in my laptop.... 😖